distribution/tools/plugin-cli/src/test/java/org/elasticsearch/plugins/RemovePluginCommandTests.java

```diff
@@ -137,6 +147,13 @@ public void testBasic() throws Exception {
         assertRemoveCleaned(env);
     }
 
+    public void testRemoveOldVersion() throws Exception {
+        createPlugin("fake", VersionUtils.randomVersionBetween(random(), Version.CURRENT.minimumCompatibilityVersion(), Version.CURRENT));
```

Comment:
Since randomVersionBetween is inclusive, I think the top end of this range should be `VersionUtils.getPreviousVersion()`?

Reply:
I pushed a commit for this.